constellations/orion/src/router/handler.go

```diff
@@ -109,6 +109,15 @@ func (h *Handler) SetupApiEndpoints() {
 		apiAFH.DELETE("/afh/:id", controllers.DeleteAFH)
 	}
 
+	apiUserClass := h.Engine.Group("api/userclass")
+	{
+		apiUserClass.POST("/create", controllers.CreateUserClass)
+		apiUserClass.GET("/class/:classId", controllers.GetUsersByClassId)
+		apiUserClass.GET("/user/:userId", controllers.GetClassesByUserId)
+		apiUserClass.GET("/class/:classId/user/:userId", controllers.GetUserClassByUserAndClass)
+		apiUserClass.POST("/uc/:id", controllers.UpdateUserClass)
+		apiUserClass.DELETE("/uc/:id", controllers.DeleteUserClass)
+	}
```

Comment:
Add one more endpoint called:

`api/userclasses/states` to returns a list of all possible states and their associated values.
So it hsould return:
```
[
    { "name": "pending", "value": 0 },
    { "name": "accepted", "value": 1 },
    { "name": "trial", "value": 2 }
]
```

This is so the front-end people can have access to all possible states and their associated values